mobile-academy/src/main/java/org/motechproject/nms/mobileacademy/service/impl/CourseNotificationServiceImpl.java

```diff
@@ -40,7 +39,7 @@
     private static final String DELIVERY_STATUS = "deliveryStatus";
     private static final String ADDRESS = "address";
     private static final String SMS_CONTENT_PREFIX = "sms.content.";
-    private static final String SMS_DEFAULT_LANGUAGE_CODE = "50";
+    private static final String SMS_DEFAULT_LANGUAGE_CODE = "default";
```

Comment:
not sure what this means, but the property is call sms.content.default. What do you recommend we call it?